src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,88 @@
  */
 
 public class Plane {
+    private final int id;
+    private final String model;
+    private final String colour;
+    private final int numberOfPassenger;
+    private final boolean isSeaplane;
 
+    private Plane(PlaneBuilder planeBuilder) {
+        this.id = planeBuilder.id;
+        this.model = planeBuilder.model;
+        this.colour = planeBuilder.colour;
+        this.numberOfPassenger = planeBuilder.numberOfPassenger;
+        this.isSeaplane = planeBuilder.isSeaplane;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public String getColour() {
+        return colour;
+    }
+
+    public int getNumberOfPassenger() {
+        return numberOfPassenger;
+    }
+
+    public boolean isSeaplane() {
+        return isSeaplane;
+    }
+
+    @Override
+    public String toString() {
+        return "Plane{"
+               + "id=" + id
+               + ", model='" + model + '\''
+               + ", colour='" + colour + '\''
+               + ", numberOfPassenger=" + numberOfPassenger
+               + ", isSeaplane=" + isSeaplane
+               + '}';
+    }
+
+    public static class PlaneBuilder {
+        private int id;
+        private String model;
+        private String colour;
+        private int numberOfPassenger;
+        private boolean isSeaplane;
+
+        public PlaneBuilder() {
```

Comment:
do we need it? By default we already have default constructor

Reply:
no, it`s was a part of some experiments, forgot to remove it)